File: packages/java-sdk/src/main/java/dev/blindfold/sdk/models/DetectResponse.java
package dev.blindfold.sdk.models;

import com.google.gson.annotations.SerializedName;
import java.util.List;

public class DetectResponse {
    @SerializedName("detected_entities")
    private List<DetectedEntity> detectedEntities;

    @SerializedName("entities_count")
    private int entitiesCount;

    public DetectResponse() {}

    public List<DetectedEntity> getDetectedEntities() { return detectedEntities; }
    public int getEntitiesCount() { return entitiesCount; }
}

File: packages/dotnet-sdk/src/Blindfold.Sdk/Models/DetectResponse.cs
using System.Text.Json.Serialization;

namespace Blindfold.Sdk.Models;

public class DetectResponse
{
    [JsonPropertyName("detected_entities")]
    public List<DetectedEntity> DetectedEntities { get; set; } = new();

    [JsonPropertyName("entities_count")]
    public int EntitiesCount { get; set; }
}
